### PR TITLE
MINOR: fix local Docker setup

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Comment by mjsax** [comment]
This patch is actually from @cmccabe and should be attributed to him (he did the fix on my machine while helping me to fix my local setup, so it was easier that I do the PR).

Hope it's possible to change the author when merging?

**Comment by mjsax** [comment]
Retest this please.

File: tests/docker/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ ENV MIRROR="http://apache.cs.utah.edu/"
 ADD ssh /root/.ssh
 RUN chmod 600 /root/.ssh/id_rsa
 RUN apt update && apt install -y unzip wget curl jq coreutils openssh-server net-tools vim python-pip python-dev libffi-dev libssl-dev
-RUN pip install -U pip && pip install --upgrade cffi ducktape==0.6.0
+RUN pip install -U pip && pip install --upgrade cffi virtualenv pyasn1 && pip install --upgrade ducktape==0.6.0
```
Comment (ijuma) [inline tests/docker/Dockerfile:25]: For my benefit: why is ducktape listed in a separate `pip install` invocation?

Comment (mjsax) [reply tests/docker/Dockerfile:25]: \cc @cmccabe

**Comment by mjsax** [comment]
Closing this, as it is contained in #3389

**Pull request closed without merging**